.gitignore

```diff
@@ -159,7 +159,7 @@ PublishScripts/
 ## The packages folder can be ignored because of Package Restore

```

Comment:
Can you please uncomment lines 157 to 167. And delete lines 168 to 170. (I had to make these changes as we didnt have paket when I copied github's standard vs ignore file)